package/src/RNCSliderNativeComponent.web.tsx

```diff
@@ -318,8 +324,10 @@ const RCTSliderWebComponent = React.forwardRef(
       <View
         ref={containerRef}
         onLayout={({nativeEvent: {layout}}: LayoutChangeEvent) => {
+          const lowerValue = lowerLimit === minimumValue ? 0 : lowerLimit;
+          const multiplier = (upperLimit - lowerValue) / maximumValue;
           containerSize.current.height = layout.height;
-          containerSize.current.width = layout.width;
+          containerSize.current.width = layout.width * multiplier;
```

Comment:
Sliding limits definitely shouldn't affect the overall layout and/or look of the Slider. They should only prevent user from sliding above the given limits, indicating there's still some value unreachable to user.
But with this implementation we manipulate with the length of the sliding track, why?